### DISCUSSION TITLE
Trying to find some good Raspberry Pi apps for my new Raspberry Pi I'm getting for Christmas.

### DISCUSSION THREAD

**Discussion by DaBFBMShow** [community/community#147829] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

So, I'm new here just to ask one thing, what are some Raspberry Pi OS/Linux apps that try to get out everything of what the Raspberry Pi has to offer.

BTW I'm getting a Raspberry Pi Pico 2 W, so make sure the app doesn't take up much space.
Thanks!
-DaBFBMshow

Edit: BTW I want them to be repositories on Github, but you can do other websites, that's fine.
Thanks!
-DaBFBMshow

**Comment by miguelangelooscardoso** [comment]
"I want them to be repositories on Github", you can upload directly what you will develop (including documentation and images) or you can create the repo clone and do your own versioning control on you local machine (desktop), the idea of build a project specifically that matches your interest in Raspberry Pi Pico depends broadly on your interests and skills. You've gotta do your research first.
